atlasdb-workload-server-distribution/var/conf/workload-server.timelock.cassandra.yml

```diff
@@ -45,4 +45,10 @@ install:
       isolationLevel: SERIALIZABLE
     iterationCount: 100
     type: singleRowTwoCells
+  ringConfig:
+    tableConfiguration:
+      tableName: ring_test
+      isolationLevel: SERIALIZABLE
```

Comment:
I know we discussed previously this really being a snapshot isolation test, which it is, but for semantics we'll just keep it serializable for now